### DISCUSSION TITLE
Did the functionality of OFRenamer change?

### DISCUSSION THREAD

**Discussion by doughnutmaster** [DIGITALCRIMINALS/OnlyFans#270] [Q&A]
Hi:
I currently use another tool to rip the profiles, and then use this tool to reorganize/rename the files.
This process use to work until recently, where now the script will just say "renaming files.." but then not move/rename any files located in the root folder.
My config has downloading disabled, but when I re enable it for a small creator, it appears the renaming and organizing is happening on the fly.
Looking forward to your help.

**Comment by DIGITALCRIMINAL** [comment]
It should be working now

**Comment by Nick05491** [comment]
> Hi: I currently use another tool to rip the profiles, and then use this tool to reorganize/rename the files. This process use to work until recently, where now the script will just say "renaming files.." but then not move/rename any files located in the root folder. My config has downloading disabled, but when I re enable it for a small creator, it appears the renaming and organizing is happening on the fly. Looking forward to your help.

Hi @doughnutmaster , could you give an example of how you use just the OFRenamer functionality?  I also have files I had downloaded previously using another scrapper.  Old scrapper used to save all files in the same directory (not sort by API... posts, messages, stories, etc.) and didn't set the 'Date Modified' shown in Finder to match then the media had been uploaded to OF.  I'm hoping to use the 'OFRenamer' of this scrapper to organize the files I had downloaded previously.
